src/sst/elements/vanadis/inst/vzicntr.h

```diff
@@ -0,0 +1,17 @@
+
+#ifndef _H_VANADIS_ZICNTR
+#define _H_VANADIS_ZICNTR
+
+#include <utility>
+
+namespace SST::Vanadis
+{
+    // Tags to be used to with the regfile and VanadisReadCounterInstruction
+    namespace Zicntr {
+        inline constexpr std::integral_constant<uint32_t, 0> CYCLE;
```

Comment:
@feldergast Is integral_constant currently supported by our compilers?

Reply:
Should be fine.  The last update to integral_constant was in c++14 as far as I can tell.